src/utils/filePathGenerator.ts

```diff
@@ -112,17 +112,26 @@ export const filePathsFromMetadataComponent = (
 
   // lwc, aura, waveTemplate, experiencePropertyType
   if (type.strategies?.adapter === 'bundle') {
-    const mappings = new Map<string, string>([
+    const mappings = new Map<string, string | string[]>([
```

Comment:
I didn't want to mess around with the other bundles, so the only way I could get this to accept my new bundle and still pass the tests was to add the or string[].